packages/expo-image-picker/src/ImagePicker.types.ts

```diff
@@ -38,19 +38,31 @@ export type ImageInfo = {
 
 export type ImagePickerResult = { cancelled: true } | ({ cancelled: false } & ImageInfo);
 
+export type ImagePickerMultipleResult = { cancelled: true } | ({ cancelled: false; selected: ImageInfo[] });
+
+
 export type ImagePickerOptions = {
   allowsEditing?: boolean;
   aspect?: [number, number];
   quality?: number;
-  allowsMultipleSelection?: boolean;
   mediaTypes?: MediaTypeOptions;
   exif?: boolean;
   base64?: boolean;
   videoExportPreset?: VideoExportPreset;
+   allowsMultipleSelection?: boolean;
 };
 
 export type OpenFileBrowserOptions = {
   mediaTypes: MediaTypeOptions;
   capture?: boolean;
   allowsMultipleSelection: boolean;
 };
+
+
+export type ExpandImagePickerOptions =
+  | (ImagePickerOptions & { allowsMultipleSelection: true })
+  | ImagePickerOptions;
```

Comment:
Having the name of this union type sound as it was a function is confusing.
Moreover I don't see any reason to have this union type at all as `ImagePikcerOptions` already defines a property `{ allowsMultipleSelection: boolean }`.
```suggestion
```